iceoryx_hoofs/test/moduletests/test_reporting_logstream.cpp

```diff
@@ -436,6 +498,36 @@ TYPED_TEST(IoxLogStreamArithmetic_test, StreamOperator_ValueMax)
     EXPECT_THAT(this->loggerMock.logs[0].message, StrEq(convertToString(this->LogValueMax)));
 }
 
+TYPED_TEST(IoxLogStreamArithmetic_test, StreamOperator_WrappedAndImplicitelyConverted_ValueLow)
```

Comment:
Nitpick, also in every other test name.

```suggestion
TYPED_TEST(IoxLogStreamArithmetic_test, StreamOperator_WrappedAndimplicitlyConverted_ValueLow)
```